src/auth/auth.service.ts

```diff
@@ -1,10 +1,4 @@
-import {
-  HttpException,
-  HttpStatus,
-  Injectable,
-  OnModuleInit
-} from '@nestjs/common';
-import { Client, ClientRedis, Transport } from '@nestjs/microservices';
+import { HttpException, HttpStatus, Injectable, Scope } from '@nestjs/common';
```

Comment:
Codacy found an issue: ['Scope' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=5802926)